runtime-sdk/src/keymanager.rs

```diff
@@ -146,15 +170,6 @@ impl<'a> KeyManagerClientWithContext<'a> {
     }
 }
 
-impl<'a> Clone for KeyManagerClientWithContext<'a> {
```

Comment:
:information_source: this is now derived

Reply:
Yeah not sure why it wasn't before.